src/components/form/Input/index.jsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+
+export default function index({ name, type, placeholder, ref, label }) {
```

Comment:
this is called index. also you probably don't want a component called Input because bootstrap antd all have the same component
 name it more specific

Reply:
in bootstap, it's called `InputGroup` or `Form.Input`, I checked it

Reply:
you shouldnt call your components something with such a generic name like Input because it will confuse readers who will think that it comes from a library etc (this is not an Input that works everywhere. it's an Input that works only in your app)

rename file, give component a name that's not "index"